tests/test_java.py

```diff
@@ -212,16 +168,12 @@ def test_java_safemode_version_check(
     ) as profiler:
         profiler.snapshot()
 
-    assert len(caplog.records) > 0
-    message = caplog.records[0].message
-    assert "Unsupported java version 8.275" in message
+    assert "Unsupported java version" in caplog.text
```

Comment:
Why? Because the local Java version differs?

Reply:
Do we want to extract the exact version as done with `jvm_version = parse_jvm_version(profiler._get_java_version(process))`

Reply:
Yes